hudi-common/src/main/java/org/apache/hudi/common/fs/StorageSchemes.java

```diff
@@ -62,6 +62,8 @@
   OBS("obs", false),
   // Kingsoft Standard Storage ks3
   KS3("ks3", false),
+  // JuiceFileSystem
+  JFS("jfs", true),
```

Comment:
can we also add some docs on how to use jfs like this http://hudi.apache.org/docs/cloud in another PR(on asf-site branch rather than master branch)

Reply:
ok, i will do it

Reply:
#3736